pages/dashboard.js

```diff
@@ -117,9 +119,17 @@ export default function Dashboard(props) {
       let orderedInactive = []
       let orderedActiveAgreement = []
 
+      //deep clone advertisingCards, set displayFlag for true for now
+      const newAdCards = {}
```

Comment:
It might be easier to just access the list from the props directly rather than re-creating the list.